app/lib/dashboard/service_calendars.rb

```diff
@@ -132,41 +132,5 @@ def self.visits_select_options(arm, cur_page=1)
 
       options_for_select(arr, cur_page)
     end
-
```

Comment:
Both of these methods were extracted into partials that could be rerendered on update.